Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19 AS builder
+FROM golang:1.19-bullseye AS builder
```

Comment:
is this `-bullseys` necessary?
If e2e can run successfully without adding this, I suggest not adding it so that there is no need to change gitlab.

Reply:
no, the pods cannot run using the old base image after verifed.